python.json

```diff
@@ -70,5 +70,29 @@
   {"id": "py.django-suit", "imports": ["suit"], "name": "Django Suit", "repo": "darklow/django-suit", "tags": ["django-admin"], "tech": ["app-blocks"]},
   {"id": "py.django-xadmin", "imports": ["xadmin"], "name": "Xadmin", "repo": "sshwsfc/xadmin", "tags": ["django-admin"], "tech": ["app-blocks"]},
   {"id": "py.flask-admin", "imports": ["flask_admin"], "name": "Flask-Admin", "repo": "flask-admin/flask-admin", "tags": ["flask-admin"], "tech": ["app-blocks"]},
-  {"id": "py.django-grappelli", "imports": ["grappelli"], "name": "Django Grappelli", "repo": "sehmaschine/django-grappelli", "tags": ["django-admin"], "tech": ["app-blocks"]}
+  {"id": "py.django-grappelli", "imports": ["grappelli"], "name": "Django Grappelli", "repo": "sehmaschine/django-grappelli", "tags": ["django-admin"], "tech": ["app-blocks"]},
+
+  {"id": "py.pypattyrn", "imports": ["pypattyrn"], "name": "PyPattyrn", "repo": "tylerlaberge/PyPattyrn", "tags": ["patterns"], "tech": ["app-arch"]},
+  {"id": "py.python-patterns", "imports": ["behavioral", "creational", "dft", "fundamental", "other", "structural"], "name": "python-patterns", "repo": "faif/python-patterns", "tags": ["patterns"], "tech": ["app-arch"]},
+  {"id": "py.sorted-containers", "imports": ["sortedcontainers"], "name": "Sorted Containers", "repo": "grantjenks/python-sortedcontainers", "tags": ["containers"], "tech": ["data-structures"]},
+  {"id": "py.scoop", "imports": ["scoop"], "name": "SCOOP", "repo": "soravux/scoop", "tags": ["concurrent-ops"], "tech": ["faster-code"]},
+
+  {"id": "py.django-simple-captcha", "imports": ["captcha"], "name": "Django Simple Captcha", "repo": "mbi/django-simple-captcha", "tags": ["captcha"], "tech": ["app-blocks"]},
+  {"id": "py.django-simple-spam-blocker", "imports": [""], "name": "Django Simple Spam Blocker", "repo": "moqada/django-simple-spam-blocker", "tags": ["spam-blocker"], "tech": ["app-blocks"]},
+
+  {"id": "py.audiolLazy", "imports": ["auidolazy"], "name": "AudioLazy", "repo": "danilobellini/audiolazy", "tags": ["real-time-dsp"], "tech": ["computational-science"]},
+  {"id": "py.audioread", "imports": ["audioread"], "name": "audioread", "repo": "beetbox/audioread", "tags": ["audio-files"], "tech": ["audio"]},
+  {"id": "py.beets", "imports": ["beets"], "name": "Beats", "repo": "beetbox/beets", "tags": ["toolkit"], "tech": ["audio"]},
+  {"id": "py.dejavu", "imports": ["dejavu"], "name": "dejavu", "repo": "worldveil/dejavu", "tags": ["audio-recognition"], "tech": ["audio"]},
+  {"id": "py.django-elastic-transcoder", "imports": ["dj_elastictranscoder"], "name": "Django Elastic Transcoder", "repo": "StreetVoice/django-elastic-transcoder", "tags": ["aws-transcoder"], "tech": ["cloud"]},
```

Comment:
tags: aws-elastic-transcoder?

Reply:
I guess either works, but it doesn't really matter. ID is not displayed anywhere.

Reply:
The comment was about tags, not ID. Just thinking if 'elastic' keyword is important here